iroha/test_network/tests/simeragi_with_mock.rs

```diff
@@ -47,7 +47,7 @@ pub mod utils {
 
```

Comment:
Can you rename this file to `sumeragi_with_mock.rs`? ;)

Reply:
Duplicate i guess :)
Closing this one, for easier tracking